packages/@okta/vuepress-site/docs/reference/api/iam-roles/index.md

```diff
@@ -0,0 +1,3027 @@
+    },
+    "resource-set": {
+      "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g"
+    }
+  }
+}
+```
+
+#### List resources
+<ApiLifecycle access="beta" />
+
+<ApiOperation method="get" url="/api/v1/iam/resource-sets/${resourceSetId}/resources" />
+
+List resources that make up a Resource Set
+
+##### Request parameters
+
+| Parameter   | Description                                                                       | Param Type   | DataType     | Required |
+| :------------- | :------------------------ | :----------- | :----------- | :------- |
+| resourceSetId  | `id` of the Resource Set  | URL          | String       | TRUE     |
+
+##### Response parameters
+
+A paginated array of [resources](#resource-object)
+
+##### Request example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g/resources"
+```
+
+##### Response example
+
+```json
+{
+  "resources": [
+    {
+      "id": "ire106sQKoHoXXsAe0g4",
+      "_links": {
+        "self": {
+          "href": "https://${yourOktaDomain}/api/v1/groups/00guaxWZ0AOa5NFAj0g3"
+        }
+      }
+    },
+    {
+      "id": "ire106riDrTYl4qA70g4",
+      "_links": {
+        "self": {
+          "href": "https://${yourOktaDomain}/api/v1/groups/00gu67DU2qNCjNZYO0g3/users"
+        }
+      }
+    },
+    {
+      "id": "irezvo4AwE2ngpMw40g3",
+      "_links": {
+        "self": {
+          "href": "https://${yourOktaDomain}/api/v1/users"
+        }
+      }
+    },
+    {
+      "id": "irezvn1ZZxLSIBM2J0g3",
+      "_links": {
+        "self": {
+          "href": "https://${yourOktaDomain}/api/v1/groups"
+        }
+      }
+    }
+  ],
+  "_links": {
+    "next": {
+      "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g/resources?after=irezvn1ZZxLSIBM2J0g3"
+    },
+    "resource-set": {
+      "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g"
+    }
+  }
+}
+```
+
+#### Delete a resource
+<ApiLifecycle access="beta" />
+
+<ApiOperation method="delete" url="/api/v1/iam/resource-sets/${resourceSetId}/resources/${resourceId}" />
+
+Remove a resource from a Resource Set
+
+##### Request parameters
+
+| Parameter      | Description                             | Param Type   | DataType     | Required |
+| :------------- | :-------------------------------------- | :----------- | :----------- | :------- |
+| resourceSetId  | id of the Resource Set                  | URL          | String       | TRUE     |
+| resourceId     | id of the resource within Resource Set  | URL          | String       | TRUE     |
+
+`resourceId` is the id obtained when [listing resources within Resource Set](#list-resources). For example, if the resource object was:
```

Comment:
id > ID
when listing resources within Resource Set. > when resources are listed within the Resource Set.